### PR TITLE
clarifications about how many credentials can be issued using the same AT

### PR DESCRIPTION
**Pull request by peppelinux**
This PR aims to resolve part of the comments shared in the issue https://github.com/openid/OpenID4VCI/issues/313

### CODE REVIEW

File: openid-4-verifiable-credential-issuance-1_0.md [hunk 1]
```diff
@@ -698,7 +698,7 @@ The Credential Endpoint issues a Credential as approved by the End-User upon pre
 
 Communication with the Credential Endpoint MUST utilize TLS. 
 
-The Client can request issuance of a Credential of a certain type multiple times, e.g., to associate the Credential with different public keys/Decentralized Identifiers (DIDs) or to refresh a certain Credential.
+The Client can request issuance of a Credential of a certain type multiple times, e.g., to associate the Credential with different public keys/Decentralized Identifiers (DIDs) or to refresh a certain Credential. It is at the Credential Issuer's discretion how many copies of a Credential it is willing to issue, how often, and whether reissued Credentials automatically update if the underlying claims have changed.
```
Comment (Sakurann) [inline openid-4-verifiable-credential-issuance-1_0.md:701]: err, PR #293 here on [line 1399](https://github.com/openid/OpenID4VCI/pull/293/files#diff-1f424614b35a9899813079f1b1f6218631a2aedd993368ccb89bb81a9eda0289R1399) introduces batch_size, so there is a way for the issuer to communicate how many credentials it is willing to issue in a batch, so it is a little more than "at the discretion of the issuer".

also I am not sure what "automatically update" means. there is no way for the issuer to push into the wallet an updated credential, the wallet need to talk to the credential endpoint explicitly to get credentials refreshed.

**Review by Sakurann — CHANGES_REQUESTED** [review]
per comments here https://github.com/openid/OpenID4VCI/pull/326/files#r1613666445

File: openid-4-verifiable-credential-issuance-1_0.md (hunk 1 above)
Comment (peppelinux) [reply openid-4-verifiable-credential-issuance-1_0.md:701]: I would be ok with this

````
The Client can request the issuance of a Credential of a certain type multiple times, for example, to associate the Credential with different public keys or Decentralized Identifiers (DIDs), or to refresh a certain Credential. However, it is important to note that the ability to do so depends on whether the Issuer supports the multiple issuance of the same Credential Type or the refresh of the Credential.
````

Comment (paulbastian) [reply openid-4-verifiable-credential-issuance-1_0.md:701]: I would rather prefer this as a suggestion in PR#293 to avoid the merge conflict.

Comment (peppelinux) [reply openid-4-verifiable-credential-issuance-1_0.md:701]: yes we can do this, see: https://github.com/openid/OpenID4VCI/pull/293#discussion_r1632079784

Comment (pmhsfelix) [reply openid-4-verifiable-credential-issuance-1_0.md:701]: Is this PR still relevant?

**Comment by jogu** [comment]
I believe this was resolved in PR #293 as per https://github.com/openid/OpenID4VCI/pull/293#discussion_r1632079784 - closing.

**Pull request closed without merging**